data/transactions/logic/assembler.go

```diff
@@ -643,6 +644,24 @@ func assembleByte(ops *OpStream, spec *OpSpec, args []string) error {
 	return nil
 }
 
+// method "add(uint64,uint64)uint64"
+func assembleMethod(ops *OpStream, spec *OpSpec, args []string) error {
+	if len(args) == 0 {
+		return ops.error("method requires a literal argument")
+	}
+	arg := args[0]
+	if len(arg) > 1 && arg[0] == '"' && arg[len(arg)-1] == '"' {
+		val, err := parseStringLiteral(arg)
+		if err != nil {
+			return ops.error(err)
```

Comment:
nit: convert `fmt.Errorf("no quotes")` in `parseStringLiteral` to a typed `QuotesError` type and handle it here.